atc/policy/checker.go

```diff
@@ -0,0 +1,160 @@
+package policy
+
+import (
+	"fmt"
+	"strings"
+
+	"code.cloudfoundry.org/lager"
+	"github.com/jessevdk/go-flags"
+)
+
+const ActionUsingImage = "UsingImage"
+
+type PolicyCheckNotPass struct{}
+
+func (e PolicyCheckNotPass) Error() string {
+	return "policy check not pass"
+}
+
+type Filter struct {
+	HttpMethods   []string `long:"policy-check-filter-http-method" description:"API http method to go through policy check"`
+	Actions       []string `long:"policy-check-filter-action" description:"Actions in the list will go through policy check"`
+	ActionsToSkip []string `long:"policy-check-filter-action-skip" default:"UsingImage" description:"Actions the list will not go through policy check"`
+}
+
+func (f Filter) normalize() Filter {
```

Comment:
I think @jamieklassen was right about this in https://github.com/concourse/concourse/pull/5034#discussion_r373024220 - `twentythousandtonnesofcrudeoil` already configures `,` as the env delimiter so this shouldn't be needed:

https://github.com/vito/twentythousandtonnesofcrudeoil/blob/3b21ad808fcb867be679c4cad408a769a3977a1b/environment.go#L21-L23

Reply:
Oh, yes. I just verified. Good to know.